### ISSUE TITLE
how do you use the template for two different vaults?

### ISSUE THREAD

**Issue by twhite96** [oleeskild/obsidian-digital-garden#370]
I tried to add a second one and it errors with `loading publication status`.

**Comment by twhite96** [comment]
> Hi @twhite96 I am actually using only 1 vault but publishing to two digital gardens. Is this what you are looking for? If so, I will write about some simple steps to make it work.

Yes! That's interesting and I'd like to know how you did it.

Thank you!

**Comment by GJRobert** [comment]
Alright, here you go: https://imazingrace.vercel.app/Obsidian/DG/multiple/
Feel free to tell if any questions. Hope gets well with yours.

**Comment by oyes77** [comment]
This should be added to the documentation as a feature OwO

**Comment by twhite96** [comment]
> This should be added to the documentation as a feature OwO

Absolutely. Or find a way to be able to use the plugin on two different vaults, which right now, is not possible in Obsidian. Hopefully soon.

**Comment by GJRobert** [comment]
@twhite96 If we change the title of this issue to "How do you publish to two different digital gardens with one vault?"
Is this what you were asking for?

**Comment by julesvirallinen** [comment]
Wrote a quick guide on how to publish from two vaults (currently unsupported and dangerous, but works if you are careful :) ) https://github.com/oleeskild/obsidian-digital-garden/discussions/415

**Comment by jonathanhung** [comment]
@julesvirallinen Hi Jules - have you tested to see what "Publish Multiple Notes" results in? 

I've been developing my digital garden on a test vault and I'm trying to migrate my "real vault" over and I'm not sure what the best procedure is for this. Was considering using your method and then just deleting the primary vault. I'm curious what "Publish Multiple" will do if I work like this... will test it out.
